client/app/account/billing/confirmTerminate/billing-confirmTerminate.service.js

```diff
@@ -1,50 +1,56 @@
-angular.module('Billing.services').service('BillingTerminate', function ($q, OvhHttp) {
-  this.getServiceTypeFromPrefix = function (serviceApiPrefix) {
-    return serviceApiPrefix
-      .replace(/^\//, '')
-      .replace(/\/\{.+$/, '')
-      .replace(/\//g, '_');
-  };
-
-  this.getServiceApi = function (serviceId, forceRefresh) {
-    const params = {
-      rootPath: 'apiv6',
-      cache: 'billingTerminateService',
+angular
```

Comment:
Seems like you can turn this into a class, do you mind doing it ? 😄

Reply:
I did not understand it. What do you mean by turning it to a class?

Reply:
He mean that you can refactor this service into a ES6 class. But that's not a requirement.